modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1663,13 +1663,13 @@ public FeatureTypeStyle transformInternal(MBStyle styleContext) {
         Fill fill = sf.fill(null, textColor(), textOpacity());
 
 
-        //Font not supported until glyphs are working
-        Font font = sb.createFont(ff.literal("" /* transformer.getDefaultFonts() */), ff.literal("normal"),
-                ff.literal("normal"), textSize());
-        /* else {
-            font = sb.createFont(ff.literal(getTextFont()), ff.literal("normal"),
-                    ff.literal("normal"), textSize());
-        } */
+        Font font = sb.createFont(ff.literal(""), ff.literal("normal"), ff.literal("normal"), textSize());
+        if (getTextFont() != null) {
```

Comment:
Yep, that makes sense. For fonts as functions, do we know if the font array as a whole can be a function or just individual entries?

If the latter, this should be fairly simple to change once we start supporting functions for fonts. Otherwise, it could be a bit more difficult.